openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -570,6 +570,15 @@
     cursor: pointer;
   }
 
+  .openassessment_container_remove_button.is--disabled,
+  .openassessment_container_remove_button.is--disabled:hover,
+  .openassessment_container_add_button.is--disabled,
+  .openassessment_container_add_button.is--disabled:hover {
+    color: $edx-gray-l2;
+    background-color: transparent;
+    cursor: auto;
+  }
+
```

Comment:
I'm not familiar with exactly where this would go, having not worked with the ORA2 styling, but for now this location is ok.